client/coral-admin/src/actions/comments.js

```diff
@@ -14,30 +14,30 @@ export const fetchModerationQueueComments = () => {
     dispatch({type: commentTypes.COMMENTS_MODERATION_QUEUE_FETCH_REQUEST});
 
     return Promise.all([
-      coralApi('/queue/comments/pending'),
+      coralApi('/queue/comments/premod'),
       coralApi('/queue/users/pending'),
       coralApi('/queue/comments/rejected'),
```

Comment:
Good thought. We have stories coming up (pagination, more data in the queues, loading counts from count edges of the graph) that will refactor this heavily. Let's address this issue in a future pr.

Ultimately, the current mod queues